colibris/views.py

```diff
@@ -1,17 +1,195 @@
+import abc
+import traceback
+from json import JSONDecodeError
 
 from aiohttp import web
 from aiohttp_apispec import docs
+from aiohttp_apispec import response_schema, request_schema
 
-from colibris import app
+from colibris.persist import Model
+from colibris.schemas import ModelSchema
+from marshmallow import ValidationError
+from peewee import IntegrityError
+
+from colibris import app, api
 from colibris.conf import settings
 
 
 async def home(request):
     raise web.HTTPFound(settings.API_DOCS_PATH)
 
 
+# TODO: I would suggest to move it into some template.
```

Comment:
Such as?

Reply:
Skeleton or any other template. I would prefer to not have such views in the core. I think (not 100% sure yet) we should only have "abstract" views in core.

Reply:
@lucifurtun agreed. Let's move it to skeleton then!